README.md

```diff
@@ -44,6 +44,7 @@ services:
       - SPOTIFY_USERNAME=
       - SPOTIFY_PASSWORD=
       - DISCORD_USER_ID=     # Discord user ID of the user you want Aoede to follow
+      - SPOTIFY_BOT_AUTOPLAY=
```

Comment:
a comment explaining what this does would be great

Reply:
Added